src/components/ArtifactDetailPage/ArtifactDetailPage.tsx

```diff
@@ -0,0 +1,139 @@
+              {serviceContainerArtifact.data?.importDate &&
+                createDateTime({ date: serviceContainerArtifact.data.importDate }).custom}
+            </AttributesItem>
+          </Attributes>
+        </ContentBox>
+      </GridItem>
+
+      <GridItem xl={12} xl2={6}>
+        <Toolbar>
+          <ToolbarItem>
+            <TextContent>
+              <Text component={TextVariants.h2}>Checksums</Text>
+            </TextContent>
+          </ToolbarItem>
+        </Toolbar>
+        <ContentBox borderTop padding contentBoxHeight={`calc(100% - 78px)`}>
+          <Attributes>
+            <AttributesItem title={artifactEntityAttributes.md5.title} forceStringWrap>
+              {serviceContainerArtifact.data?.md5}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.sha1.title} forceStringWrap>
+              {serviceContainerArtifact.data?.sha1}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.sha256.title} forceStringWrap>
+              {serviceContainerArtifact.data?.sha256}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.build.title}>
+              {serviceContainerArtifact.data?.build && (
+                <>
+                  <BuildName build={serviceContainerArtifact.data.build} long includeBuildLink includeConfigLink /> (
+                  <Link to={`/builds/${serviceContainerArtifact.data.build.id}`}>#{serviceContainerArtifact.data.build.id}</Link>)
+                </>
+              )}
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.buildCategory.title}>
+              <Label color="grey">{serviceContainerArtifact.data?.buildCategory}</Label>
+            </AttributesItem>
+          </Attributes>
+        </ContentBox>
+      </GridItem>
+
+      <GridItem span={12}>
+        <Toolbar>
+          <ToolbarItem>
+            <TextContent>
+              <Text component={TextVariants.h2}>Locations</Text>
+            </TextContent>
+          </ToolbarItem>
+        </Toolbar>
+        <ContentBox borderTop padding>
+          <Attributes>
+            <AttributesItem title={artifactEntityAttributes.publicUrl.title} forceStringWrap>
+              <DownloadLink url={serviceContainerArtifact.data?.publicUrl} title={serviceContainerArtifact.data?.publicUrl} />
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.originUrl.title} forceStringWrap>
+              <DownloadLink url={serviceContainerArtifact.data?.originUrl} title={serviceContainerArtifact.data?.originUrl} />
+            </AttributesItem>
+            <AttributesItem title={artifactEntityAttributes.deployUrl.title} forceStringWrap>
+              <DownloadLink url={serviceContainerArtifact.data?.deployUrl} title={serviceContainerArtifact.data?.deployUrl} />
+            </AttributesItem>
+          </Attributes>
```

Comment:
Hi @matedo1  , shall we add some tooltips for Public Url, Origin Url, and Deploy Url to explain more to our users(especially new users)? If yes, could you provide some guide definitions for these URLs? Thanks!

Reply:
Yes, we can add them, but please try to reach someone from backend to provide the best phrasing for them.